app/controllers/usermatching/HomeController.scala

```diff
@@ -105,6 +133,38 @@ class HomeController @Inject()(val auditingService: AuditingService,
     }
   }
 
+  private def populateSubscription(reference: String, prePopData: PrePopData)(implicit request: Request[AnyContent]): Future[Unit] = {
+    // Set up futures so that they parallelise.
+    val futureSaveSelfEmployments = prePopData.selfEmployments
+      .map(sse => sse.map(
+        se => SelfEmploymentData(
+          UUID.randomUUID().toString,
+          None,
```

Comment:
```suggestion
          se.businessStartDate,
```
Why is the business start date not set here?

Reply:
Missed it.  Done.